src/ASBNApp.DataAPI/Program.cs

```diff
@@ -0,0 +1,74 @@
+using ASBNApp.DataAPI.Context;
+using ASBNApp.DataAPI.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.OData;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OData.ModelBuilder;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Identity stuff
+builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme).AddIdentityCookies();
+builder.Services.AddAuthorizationBuilder();
+builder.Services.AddIdentityCore<User>()
+	.AddEntityFrameworkStores<ASBNAppContext>()
+	.AddApiEndpoints();
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
```

Comment:
just be aware this will not anymore support with dotnet 9 and aboth. Read more about it here:

https://github.com/dotnet/aspnetcore/discussions/58103